desktop-app/app/Routes.js

```diff
@@ -1,5 +1,4 @@
 import React, {Fragment} from 'react';
```

Comment:
@manojVivek I think we should rename this as `App.js`, move the app component from `Root.js` here. That would be much cleaner in two ways, `Root.js` will have only one component and `Routes.js` doesn't make any sense since we have removed routing.

Reply:
Yes this makes sense, please do it.

Reply:
Tried to do it, but `App` component uses a hook and `Root` is a class component. This will lead to further refactor beyond the scope of this PR. So renamed `Routes.js` to `AppContent.js` for now.

Reply:
ok, that's cool!